README.md

```diff
@@ -81,7 +86,14 @@ Mimic the site's functionality around Users and Voting:
 - Add a list of voting users to the details page for each media
 - Don't allow a user to vote for the same media more than once
 - Add a page for each user, as well as a page showing a summary of all users
-- Make sure any models you've built so far are well-tested
```

Comment:
If we're asking for tests around voting logic, it might be wise to explicitly state that this should be implemented in the model.